torch/_inductor/codegen/triton.py

```diff
@@ -2036,6 +2040,15 @@ def store(
         elif mode is None:
             line = f"tl.store({var} + ({indexing.index_str}), {value}, {indexing.mask_str})"
         elif mode == "atomic_add":
+            if (
+                isinstance(index, sympy.core.symbol.Symbol)
+                and symbol_is_type(index, SymT.TEMPLATE_INDEX)
+                or "idx" in indexing.index_str
```

Comment:
We often have expressions of our index variables and therefore the symbol is type does not match correctly..